src/views/Items/Item.js

```diff
@@ -0,0 +1,43 @@
+import React, {Component, PropTypes} from 'react';
+import classNames from 'classnames';
+
+export default class Item extends Component {
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+    description: PropTypes.node,
+    extra: PropTypes.node,
+    heading: PropTypes.node,
+    image: PropTypes.node,
+    meta: PropTypes.node,
+  };
+
+  render() {
+    let heading = <div className='header'>{this.props.heading}</div>;
+    let meta = <div className='meta'>{this.props.meta}</div>;
+    let description = <div className='description'>{this.props.children || this.props.description}</div>;
```

Comment:
By default, will render children before the default property if they both exist on the component.